SofaKernel/modules/SofaExplicitOdeSolver/EulerSolver.cpp

```diff
@@ -59,6 +59,7 @@ SOFA_DECL_CLASS(Euler);
 
 EulerSolver::EulerSolver()
     : symplectic( initData( &symplectic, true, "symplectic", "If true, the velocities are updated before the positions and the method is symplectic (more robust). If false, the positions are updated before the velocities (standard Euler, less robust).") )
+    , d_threadsafevisitor(initData(&d_threadsafevisitor, false, "threadsafevisitor", "If true, do not use realloc and free visitors in fwdInteractionForceField."))
```

Comment:
just a quick remark, to follow Sofa coding rules. Could you replace the `d_threadsafevisitor ` by `d_threadsafeVisitor`  or `d_threadSafeVisitor` not sure which one is the most logical.

Reply:
same in the other Solver classes

Reply:
done commit 72d9ea24f2972f803113ba142599b391be429580.